asserts/database.go

```diff
@@ -99,28 +92,11 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 	bs := cfg.Backstore
 	keypairMgr := cfg.KeypairManager
 
-	// falling back to at least one of the filesytem backstores,
-	// ensure the main directory cfg.Path
-	// TODO: decide what should be the final defaults/fallbacks
-	if bs == nil || keypairMgr == nil {
-		err := os.MkdirAll(cfg.Path, 0775)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create assert database root: %v", err)
-		}
-		info, err := os.Stat(cfg.Path)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create assert database root: %v", err)
-		}
-		if info.Mode().Perm()&0002 != 0 {
-			return nil, fmt.Errorf("assert database root unexpectedly world-writable: %v", cfg.Path)
-		}
-
-		if bs == nil {
-			bs = newFilesystemBackstore(cfg.Path)
-		}
-		if keypairMgr == nil {
-			keypairMgr = newFilesystemKeypairMananager(cfg.Path)
-		}
+	if bs == nil && keypairMgr == nil {
```

Comment:
I'd be okay with panicking right here, but it doesn't look like we do that. It's also strange that we error when both are nil but not when one is nil. Why don't we panic consistently if we find either of them nil then?